scripts/components/Scene/SceneTypes/StaticScene.js

```diff
@@ -59,16 +77,42 @@ export default class StaticScene extends React.Component {
    * React life-cycle handler: Component did update.
    */
   componentDidUpdate() {
+    // Remove wheel event listener from prev scene
+    if (!this.props.isActive && this.props.sceneId === this.props.sceneWaitingForLoad) {
+      this.sceneWrapperRef.current?.removeEventListener('wheel', this.handleMouseWheel.bind(this), false);
```

Comment:
When we bind `this` to a function, we create a new function reference. Therefore, we don't remove the same reference that we added.

Reply:
Better now?

Reply:
Yes! 💐